lib/modules/datasource/datasource.ts

```diff
@@ -25,6 +26,12 @@ export abstract class Datasource implements DatasourceApi {
 
   registryStrategy: RegistryStrategy | undefined = 'first';
 
+  releaseTimestampSupport = false;
+  releaseTimeStampNote: string | undefined;
```

Comment:
```suggestion
  releaseTimeStampNote?: string | undefined;
```

Reply:
Does it need the `| undefined` part? If so, what's the difference?

Reply:
It is effectively the same, I like simply to be more verbose when in doubt.